### PR TITLE
Add support for ```google_app_engine_standard_app_version``` to TGC

### PR DESCRIPTION
**Pull request by dfdossantos**
adding support for appengine.googleapis.com/Version

```release-note:none
```

### CODE REVIEW

File: mmv1/third_party/tgc/tests/data/example_google_app_engine_standard_app_version.tf [hunk 1]
```diff
@@ -0,0 +1,33 @@
+terraform {
+  required_providers {
+    google = {
+      source = "hashicorp/google-beta"
```
Comment (SarahFrench) [inline mmv1/third_party/tgc/tests/data/example_google_app_engine_standard_app_version.tf:4]: 👋 Hi!

Should this be `google-beta`, or `google`? The current error in this PR's checks mentions that google-beta might be a problem:

```
        logger.go:130: 2024-06-26T14:54:56.599Z	DEBUG	google_app_engine_application.my_app_v1: resource uses the google-beta provider and may not be convertible
        logger.go:130: 2024-06-26T14:54:56.599Z	DEBUG	google_app_engine_standard_app_version.my_app_v1: resource uses the google-beta provider and may not be convertible
```

Comment (dfdossantos) [reply mmv1/third_party/tgc/tests/data/example_google_app_engine_standard_app_version.tf:4]: > 👋 Hi!
> 
> Should this be `google-beta`, or `google`? The current error in this PR's checks mentions that google-beta might be a problem:
> 
> ```
>         logger.go:130: 2024-06-26T14:54:56.599Z	DEBUG	google_app_engine_application.my_app_v1: resource uses the google-beta provider and may not be convertible
>         logger.go:130: 2024-06-26T14:54:56.599Z	DEBUG	google_app_engine_standard_app_version.my_app_v1: resource uses the google-beta provider and may not be convertible
> ```

All the others supports added for TGC was done with the google-beta one, so I don't think this should be a problem

**Review by SarahFrench — APPROVED** [review]
Checks pass - LGTM

**Pull request merged**